packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -474,6 +498,11 @@ class NoDefaultCupertinoThemeData {
   /// Defaults to [CupertinoColors.systemBackground].
   final Color? scaffoldBackgroundColor;
 
+  /// Color of the selection handle.
+  ///
+  /// Defaults to [CupertinoColors.systemBlue].
+  final Color? selectionHandleColor;
```

Comment:
nit: Let's use some of the docs from here:

https://github.com/flutter/flutter/blob/adae8bbdbaed53ef305726fcfe811b2351d73a1a/packages/flutter/lib/src/material/text_selection_theme.dart#L49-L52

we can exclude the ending iOS bit.